apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -56,82 +56,25 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.ELEMENTARY
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.ELEMENTARY);
-            }}
-          />
-          <Button
-            id={'uitest-middle-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsMiddle()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.MIDDLE
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.MIDDLE);
-            }}
-          />
-          <Button
-            id={'uitest-high-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsHigh()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.HIGH
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.HIGH);
-            }}
-          />
-          <Button
-            id={'uitest-hoc-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseOfferingHOC()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.HOC
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.HOC);
-            }}
-          />
-          <input
-            checked={decideLater}
-            className={moduleStyles.input}
-            type="checkbox"
-            id="decide-later"
-            onChange={toggleDecideLater}
-          />
-          <label className={moduleStyles.decideLater} htmlFor="decide-later">
-            {i18n.decideLater()}
-          </label>
-        </div>
+      <div className={moduleStyles.input}>
+        <label className={moduleStyles.decideLater} htmlFor="decide-later">
+          {i18n.decideLater()}
+        </label>
+        <input
+          checked={decideLater}
+          className={moduleStyles.inputBox}
+          type="checkbox"
+          id="decide-later"
+          onChange={toggleDecideLater}
+        />
+        <h3>{i18n.assignACurriculum()}</h3>
+        <h5>{i18n.useDropdownMessage()}</h5>
       </div>
+      <CurriculumQuickAssignTopRow
+        isPl={false}
+        marketingAudience={marketingAudience}
+        updateMarketingAudience={updateMarketingAudience}
```

Comment:
I'm reusing the function already written, but just passing it down as a prop instead so the child component has a way to update which table renders from here